src/components/Form/Form.js

```diff
@@ -57,7 +62,7 @@ export function Form({ children, config }) {
     // If we wanted to have a form be optional, you could have its initial
     // value set to empty string, so the form could be submitted without
     // the field being touched.
-    initialValues[x] = `Initial invalid state for ${x}`
+    initialValues[fieldName] = `Initial invalid state for ${fieldName}`
```

Comment:
This is essentially what makes it all work really. @kenning just kinda going along with your thing here :)